README.md

```diff
@@ -13,21 +12,38 @@ Teams spend significant time building parts of the design system (colors, variab
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopinionated
+There are a million and one great ways to build your next website or application. To support the technologies you choose, creating unopinionated code is at the center of every decision we make in FAST-DNA.
+
+This principle of being unopinionated manifests in several important ways including:
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your styling technology of choice, without having to re-build or duplicate the base components.
+* Framework agnostic tooling that lets you use our development tools with any view framework.
+* The ability to replace almost any FAST-DNA package with your package of choice. Get started with our animation or layout package and add more as you need them. Alternatively, use our suite of packages to build your next project from the ground up; it's your call.
+
+### UI development and style guide tools
+When developing components and views, excellent development tooling can make all the difference. FAST-DNA offers development and style guide tools that work with FAST components, components from other frameworks, or your components. 
 
-## Browser support
-Cross browser testing based on web browser rendering engines.
+Try out component properties with an auto-generated props UI, get a live preview of the code based on any property’s configuration, preview localization (RTL/LTR) and themes, and preview component compositing with a transparency grid.
 
-![Browser matrix](https://saucelabs.com/browser-matrix/FAST-DNA.svg)
+Also, we built FAST development tools from re-usable packages so, if you have special needs, you can build your tools from the same shared libraries.
 
-[![Build Status](https://saucelabs.com/buildstatus/FAST-DNA)](https://saucelabs.com/beta/builds/33e17283a9fb447ab220a8e00abe8f26)
+### Bring your design system
+Widely available design systems from companies like Microsoft (Fluent), Google (Material), or Salesforce (Lightning) are useful when it is essential to align with a platform or take advantage of a polished system at low cost, but many companies have their design system of their own, and some may have multiple design systems or variations.
+
+Because FAST-DNA has abstracted base components from their style, you get a head start on your design system by building on top of tried and true base components and style libraries like our offerings for color, animation, and elevation.
+
+## Packages
+FAST-DNA is a mono repository managed with [Lerna](https://github.com/lerna/lerna#independent-mode---independent). Users choose à la carte which package to use in their project.
```

Comment:
`#independent-mode---independent` anchor isn't valid anymore, nor do I think it's really necessary to point to a specific heading when talking about Lerna generally - I'm in favor of removing and linking simply to `https://github.com/lerna/lerna`

Reply:
Fixed.